.travis.yml

```diff
@@ -1,6 +1,17 @@
 sudo: false
 language: groovy
```

Comment:
Oh, I didn't know you could do that - I thought they had different images per language. But I'll give it a shot.